recipes/krtc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "krtc" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: b715b07881316842c359d2377222c476f5e24d77b194b7eeaeb6e741a1204d95
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [win]
```

Comment:
This can be a noarch python package

Reply:
@scopatz it can't as `pykerberos` is not available for Windows.

Reply:
Sadly, krtc won't build for windows. From my comment above:

> Noting that pykerberos is not available on windows through conda-forge, I'll make `krtc` not build for it and `psdm_qs_cli` include `krtc` on all platforms except for windows.

Do you have a better solution?

Reply:
It was my understanding that you could leave *this* package as noarch and just let the lack of dependency keep it from installing on windows.

Reply:
@dopplershift but that will create an error with the CI here at the recipe. Would that just be ignored? Seems quite wrong to show a package as available and it fails to install as dependencies are not available. Can we move forward with it not being `noarch`?

Reply:
`noarch` recipes are built only on Linux, so the failures on Windows and OSX can be ignored during the build.

Reply:
Thanks so much for clarifying, @npavlovikj. I'll do that.